docs/conf.py

```diff
@@ -12,28 +12,78 @@
 #
 import os
 import sys
+import shutil
+import subprocess
+
 import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath('../lib'))
-sys.path.insert(0, os.path.abspath('../lib/sedna'))
+try:
+    import m2r2
+except ModuleNotFoundError:
+    subprocess.check_call([sys.executable, "-m", "pip", "install", "m2r2"])
+
+try:
+    import autoapi
+except ModuleNotFoundError:
+    subprocess.check_call([sys.executable, "-m", "pip",
+                           "install", "sphinx-autoapi"])
+
+
+_base_path = os.path.abspath('..')
+sys.path.append(os.path.join(_base_path, "lib"))
+sys.path.append(_base_path)
+
+extra_paths = [
+    os.path.join(_base_path, "examples"),
+]
+for p in extra_paths:
+    dst = os.path.join(
+        _base_path, "docs",
+        os.path.basename(p)
+    )
+    if os.path.isfile(dst):
+        os.remove(dst)
+    elif os.path.isdir(dst):
+        shutil.rmtree(dst)
+    if os.path.isdir(p):
+        shutil.copytree(p, dst)
+    else:
+        shutil.copy2(p, dst)
+
+
+with open('../lib/sedna/VERSION', "r", encoding="utf-8") as fh:
```

Comment:
I don't think this module needs to be shown.

Reply:
This defines the current version of sedna, and some experts have proposed get it from `git tags`, but I haven't found a good mehod to do it. @llhuii @ugvddm